Patch out rpath hardcoding since it completely ignores --disable-rpath
and other configure ways.

Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>

diff -Nura mcrypt-2.6.8.orig/config.rpath mcrypt-2.6.8/config.rpath
--- mcrypt-2.6.8.orig/config.rpath	2013-01-07 13:05:22.626883480 -0300
+++ mcrypt-2.6.8/config.rpath	2013-01-07 13:12:47.196090608 -0300
@@ -153,7 +153,7 @@
   # here allows them to be overridden if necessary.
   # Unlike libtool, we use -rpath here, not --rpath, since the documented
   # option of GNU ld is called -rpath, not --rpath.
-  hardcode_libdir_flag_spec='${wl}-rpath ${wl}$libdir'
+  hardcode_libdir_flag_spec=
   case "$host_os" in
     aix3* | aix4* | aix5*)
       # On AIX/PPC, the GNU linker is very broken
